frontend/view-edit-client/view-edit-demographics-info.component.tsx

```diff
@@ -146,6 +194,61 @@ export default function ViewEditDemographicsInfo(
         return `Unknown employment status`;
     }
   }
+
+  function handleSubmit(evt, demographicsInfo) {
+    evt.preventDefault();
+    dispatchApiStatus({
+      type: UpdateActionType.update,
+      newClientData: {
+        civilStatus: demographicsInfo.civilStatus,
+        householdIncome: demographicsInfo.householdIncome,
+        householdSize: demographicsInfo.householdSize,
+        dependents: demographicsInfo.dependents,
+        currentlyEmployed: demographicsInfo.currentlyEmployed,
+        weeklyEmployedHours: demographicsInfo.weeklyEmployedHours,
+        employmentSector: demographicsInfo.employmentSector,
+        payInterval: demographicsInfo.payInterval,
+        countryOfOrigin: demographicsInfo.countryOfOrigin,
+        dateOfUSArrival: demographicsInfo.dateOfUSArrival,
+        homeLanguage: demographicsInfo.homeLanguage,
+        isStudent: demographicsInfo.isStudent,
+        englishProficiency: demographicsInfo.englishProficiency,
+        eligibleToVote: demographicsInfo.eligibleToVote,
+        registeredToVote: demographicsInfo.registeredToVote
+      }
+    });
+  }
+
+  function getDemographicsClient(): DemographicInformationClient {
```

Comment:
Does DemographicInformationClient  need to be changed to SingleClient here to match the change on 287

Reply:
No. There are two slightly different definitions of a client -- the one in AddClient and the one in ViewEditClient. Both need to work with DemographicInformationClient, so DemographicInformationClient has its own definition of the properties it needs in order to work.